desktop/components/auction_block/index.jsx

```diff
@@ -0,0 +1,25 @@
+import AuctionBlock from './auction_block.jsx'
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { isEmpty, isString } from 'underscore'
+
```

Comment:
One thing I would do here to make it a bit more API like is

```javascript
export {
   AuctionBlock
}
```

so that later, rather than having to import the full path from within the current auctions component, we can just do 

```javascript
import { AuctionBlock } from 'components/auction_block'`
...
```